Solaris/DmarcAggregateParser.php

```diff
@@ -80,6 +80,10 @@ function parse( $files ) {
 			}
 
 			try {
+                    		foreach (['dkim', 'spf'] as $type) {
```

Comment:
Eh? Looking at it now it looks so very wrong :D WTF have I done? I'll check it out in the office tomorrow.